src/state/assets/reducer.js

```diff
@@ -0,0 +1,106 @@
+import {
+  SET_ASSETS,
+  SET_CATEGORY_FILTER,
+  SET_ACTIVE_FILTERS,
+  REMOVE_ACTIVE_FILTER,
+  FILE_TYPE_CHANGE,
+  ASSETS_VIEW_CHANGE,
+  APPLY_SORT,
+  CHANGE_PAGINATION,
+} from 'state/assets/types';
+
+const defaultState = {
+  language: 'en',
+  filteringCategories: [],
+  activeFilters: [],
+  assetsView: 'list',
+  fileType: 'image',
+  paginationOptions: {
+    page: 1,
+    perPage: 5,
+    lastPage: 1,
+    totalItems: 0,
+    perPageOptions: [5, 10, 15, 25, 50],
+  },
+};
```

Comment:
FYI, please remove your pagination related state around your PR and use `state/pagination` for general pagination purposes. it's widely being used around app-builder and cms.